migrations/sql/V1.30.0__CE-556.sql

```diff
@@ -0,0 +1,53 @@
+--
+-- INSERT INTO drug_method_code
+--
+INSERT INTO drug_method_code (drug_method_code, short_description, long_description, display_order, active_ind, create_user_id, create_utc_timestamp, update_user_id, update_utc_timestamp)
+  VALUES('ORNA', 'Oral/nasal', 'Oral/nasal', 4, true, 'FLYWAY', now(), 'FLYWAY', now());
+
+--
+-- UPDATE Reverse distribution -> Returned to vet
+--
+UPDATE case_management.drug_remaining_outcome_code
+  SET short_description='Returned to vet', long_description='Returned to vet', update_user_id='FLYWAY', update_utc_timestamp=now()
+  WHERE drug_remaining_outcome_code='RDIS';
+
+--
+-- CREATE TABLE drug_additional_comments
+--
+
+ALTER TABLE case_management.drug_administered
+  ADD additional_comments_text varchar(1000) NULL;
```

Comment:
Our current standard for large blocks of text like this (Notes, Complaint Details, Suspect Witness details) is to use text type.

Reply:
Revised the column with cd87468.